matrix-sdk-android/src/androidTest/java/org/matrix/android/sdk/internal/crypto/E2eeSanityTests.kt

```diff
@@ -339,18 +330,15 @@ class E2eeSanityTests : InstrumentedTest {
 
         // ensure bob can now decrypt
         cryptoTestHelper.ensureCanDecrypt(sentEventIds, newBobSession, e2eRoomID, messagesText)
-
-        testHelper.signOutAndClose(newBobSession)
     }
 
     /**
      * Check that a new verified session that was not supposed to get the keys initially will
      * get them from an older one.
      */
     @Test
-    fun testSimpleGossip() {
-        val testHelper = CommonTestHelper(context())
-        val cryptoTestHelper = CryptoTestHelper(testHelper)
+    fun testSimpleGossip() = withTestHelpers(context()) { cryptoTestHelper ->
+        val testHelper = cryptoTestHelper.testHelper
```

Comment:
How to ensure that the testHelper from the cryptoTestHelper will be used in the future tests?
Make the CommonTestHelper constructor private somehow?